tests/Tests/500__jupyterhub/test.robot

```diff
@@ -59,7 +59,13 @@ Can Spawn Notebook
    Remove Spawner Environment Variable  env_five
    Remove Spawner Environment Variable  env_six
    Spawn Notebook
-   Wait for JupyterLab Splash Screen  timeout=30
+   Click Button  Access server
+   SeleniumLibrary.Switch Window  NEW
```

Comment:
wanted to understand why we need to switch windows?

Reply:
Because after clicking on `Access server` the JL interface is opened in a new tab